frontend/src/concepts/userSSAR/AccessReviewContext.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { checkAccess } from '~/api';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import useNamespaces from '~/pages/notebookController/useNamespaces';
+
+type AccessReviewCacheThingType = {
```

Comment:
Maybe we could rename this? 
```suggestion
type AccessReviewCacheStatusType = {
```

Reply:
Hah, good catch -- didn't see that

Reply:
lol thats my sloppy naming carrying over